src/components/TextArea.jsx

```diff
@@ -0,0 +1,36 @@
+function TextArea() {
+  return (
+    <div className="flex flex-col space-y-2 py-20 bg-white">
+      <h1 className="text-4xl font-bold sticky -mt-32 text-orange-500">
+        Title
+      </h1>
+
+      <p className="text-gray-600 p-10 w-3/4 mx-auto my-auto pb-4">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus
+        mattis tortor ut tortor cursus, ac maximus tortor.
+      </p>
+
+      <p className="text-green-600">HEX: #b19cd9</p>
+      <p className="text-green-600">RGB: rgb(177, 156, 217)</p>
+      <p className="text-green-600">HSL: hsl(261, 45%, 73%)</p>
+
+      <label
+        className="block text-gray-700 text-base font-bold mb-1 w-2/3 mx-auto pt-5"
+        htmlFor="description">
+        Describe where do you plan to use this color?
+      </label>
+      <textarea
+        id="description"
+        className="w-2/3 px-3 py-10 text-gray-700 border rounded-lg focus:outline-none mx-auto"
```

Comment:
I am not sure how I feel bout the padding on the text in this textarea. With text in the box it feels like I begin to type on 'line 2' although it's not.

Another thing that can happen with this textarea is it can be resized to larger than the screen to cause funky breaking stuff...
![image](https://github.com/chingu-voyages/v49-tier2-team-16/assets/33106391/a20fc335-5216-4e56-8f05-2b843b2d54b9)

You could fix this by giving it a class of `max-h-48` (change the number to change max-height)